awswrangler/catalog/_definitions.py

```diff
@@ -96,6 +97,95 @@ def _parquet_partition_definition(
         definition["StorageDescriptor"]["Columns"] = [
             {"Name": cname, "Type": dtype} for cname, dtype in columns_types.items()
         ]
+
```

Comment:
Nitpick - for some reason it looks like blank lines were added here and above (L77)